docs/helm-chart/index.rst

```diff
@@ -86,9 +86,8 @@ To install this chart using Helm 3, run the following commands:
 
 .. code-block:: bash
 
-    kubectl create namespace airflow
     helm repo add apache-airflow https://airflow.apache.org
-    helm install airflow apache-airflow/airflow --namespace airflow
+    helm upgrade airflow apache-airflow/airflow --namespace airflow --create-namespace --install
```

Comment:
```suggestion
    helm upgrade --install airflow apache-airflow/airflow --namespace airflow --create-namespace
```

nit, maybe it's my muscle memory, but tossing `--install` right after `upgrade` makes it more clear to me its "upgrade or install"